src/Common/Resource/Alias.php

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Common\Resource;
+
+/**
+ * @package OpenStack\Common\Resource
+ */
+class Alias
+{
+    /**
+     * @var bool
+     */
+    private $_isList;
+
+    /**
+     * @var string
+     */
+    private $_className;
+
+    /**
+     * @var string
+     */
+    private $_propertyName;
+
+    /**
+     * Alias constructor.
+     *
+     * @param string      $propertyName
+     * @param string|null $className
+     * @param bool        $list
+     */
+    public function __construct(string $propertyName, string $className = null, bool $list = false)
+    {
+        $this->_isList = $list;
+        $this->_propertyName = $propertyName;
+        $this->_className = (($className && class_exists($className)) ? $className : null);
+    }
+
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return $this->_propertyName;
+    }
+
+    /**
+     * @param ResourceInterface $resource
+     * @param mixed             $value
+     *
+     * @return mixed
+     */
+    public function getValue(ResourceInterface $resource, $value)
+    {
+        $class = $this->_className;
+
+        if ($value === null || !$class) {
+            return $value;
+        } elseif ($this->_isList && is_array($value)) {
```

Comment:
let's change this to `is_iterable`

Reply:
`is_iterable` available from PHP 7.1 but minimal required version for the project is 7.0